vcxagency-node/src/configuration/app-config.js

```diff
@@ -42,6 +43,10 @@ const configValidation = Joi.object().keys({
   LOG_JSON_TO_CONSOLE: Joi.string().valid(['true', 'false']),
   SERVER_PORT: Joi.number().integer().min(1025).max(65535).required(),
   SERVER_MAX_REQUEST_SIZE_KB: Joi.number().integer().min(1).max(MB_AS_KB * 10).required(),
+  SERVER_ENABLE_TLS: Joi.string().valid(['true', 'false']),
+  CERTIFICATE_PATH: Joi.string().allow(''),
```

Comment:
Not a request for change, just a question.- is there functional difference between specifying `allow('')` and omitting it?

Reply:
Do you mean ommiting the field entirely or omitting `.allow''()`? If the first, functionally probably not, but `configValidation` also serves as a documentation. If the second, [there is](https://github.com/sideway/joi/blob/master/API.md#string):

> Note that the empty string is not allowed by default and must be enabled with allow(''). Don't over think, just remember that the empty string is not a valid string by default. Also, don't ask to change it or argue why it doesn't make sense. This topic is closed.